client/src/components/Workbench/Workbench.jsx

```diff
@@ -31,7 +31,7 @@ function Workbench() {
     hideDataFiles,
   } = useSelector(
     (state) => ({
-      loading: state.workbench.loading || state.systems.storage.loading,
+      loading: state.workbench.loading,
```

Comment:
Should we use `systemsLoading` here in lieu of `state.systems.storage.loading`?

Reply:
I think you're looking at a deleted line, the `state.systems.storage.loading` is gone now that we only fetch systems once on the initial load.

Reply:
@jarosenb Yeah, I was thinking adding back in `systemsLoading` - before, the workbench would load after waiting for systems to load, and now the workbench may load before systems have loaded

Reply:
OK, I see that there's an edge case where it'd be possible to open the system selector modal before the systems are loaded in. I've added back the line to stop everything from rendering until we have systems. This kind of thing is a huge reason why I want to bring in React-Query. There are so many places where we stop the world to wait a piece of data we might not even need, where we could be declaring our data requirements at the point of use.

Reply:
Totally agree.